src/scripts/events/domEvents.js

```diff
@@ -41,8 +41,9 @@ const domEvents = () => {
       // eslint-disable-next-line no-alert
       if (window.confirm('Want to delete?')) {
         const [, firebaseKey] = e.target.id.split('--');
-        // deleteSingleAuthor(firebaseKey).then((authorsArray) => showAuthors(authorsArray));
-        deleteAuthorBooks(firebaseKey).then((showAuthors));
+        // deleteAuthorBooks(firebaseKey).then((authorsArray) => showAuthors(authorsArray));
+        deleteAuthorBooks(firebaseKey);
+        getAuthors(uid).then((authors) => showAuthors(authors));
```

Comment:
Shorthand getAuthors(uid).then(showAuthors); could also work here since .then() is a callback function and returns the response of the promise it's attached to. ✨